packages/layer-composer/src/generators/heatmap/util/time-chunks.ts

```diff
@@ -212,11 +219,11 @@ export const getActiveTimeChunks = (
   activeEnd: string,
   datasetStart: string,
   datasetEnd: string,
-  interval?: IntervalOption
+  interval?: Interval | Interval[]
 ): TimeChunks => {
   const delta = +toDT(activeEnd) - +toDT(activeStart)
   const finalInterval: Interval =
-    !interval || interval === 'auto' ? getInterval(delta) : (interval as Interval)
+    !interval || Array.isArray(interval) ? getInterval(delta, interval) : (interval as Interval)
```

Comment:
I'm in favor of removing support for `interval` as a single string - we don't need backwards compatibility as resolveDataviews will always return an array anyways (also rename it to `intervals`)

Reply:
We need to support a fixed interval for the environmental layers and moving to an array will make the logic harder so let's go as we are for now and let's keep an eye on this